js added, created footer and skills

Ashanya20 opened the pull request:
Used DevTool to check my CSS.

Ashanya20 committed:
js added, created footer and skills

juddin1995 commented on index.html:
```diff
@@ -83,6 +84,8 @@ <h2>Connect</h2>
         </ul>
     </section>
+
+    <script src="js/index.js"></script>
```
I see that the lesson says to add this script tag before the closing body tag but this solution is sub-optimal. The script will still be requested and may load before the elements on the page are rendered by the DOM causing event listeners/ handlers to break. A more elegant way to prevent this is to use the defer attribute within the script tag. Please refer to this article for further information. [Script tag defer attribute](https://javascript.info/script-async-defer).

juddin1995 commented on js/index.js:
```diff
@@ -0,0 +1,20 @@
+const foot = document.createElement('footer');
```
I think a better name for this would be footerEl or footerElement rather than foot. According to the lesson assignment rubric the variable should be named 'footer'.

juddin1995 reviewed (approved):
LGTM (Looks good to me). Great job on the stretch goal. However there are small changes that I'd highly recommend you make such as placing the script tag at the top of the document and using the defer attribute (this will ensure DOM content loads in the proper order). Also, please change the name of your 'foot' variable to 'footer', 'footerElement', or 'footerEl'. 

Another change I'd suggest is to use flex-direction column to get all your elements to stack in tablet/mobile view.  Please let me know if you have any questions.

Please add a CSS rule for the footer so that you can add some padding. The text 'Copyright' is too close to the footer's left margin.

Ashanya20 committed:
updates

Ashanya20 commented:
I updated my PR:
- Added defer to the script and moved it up in the head
- Rename foot to footerElement
- I checked: flex-direction column used in tablet/mobile view
- Added CSS rule for the footer
